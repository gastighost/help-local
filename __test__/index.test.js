// import { render, screen } from '@testing-library/react'
// import Home from '../pages/index'
// import '@testing-library/jest-dom'
// const {MongoClient} = require('mongodb');

// describe('Home', () => {
//   let connection;
//   let db;

//   beforeAll(async () => {
//     connection = await MongoClient.connect(global.__MONGO_URI__, {
//       useNewUrlParser: true,
//     });
//     db = await connection.db(global.__MONGO_DB_NAME__);
//   });

//   afterAll(async () => {
//     await connection.close();
//     if(db.close) {
//       await db.close();
//     }
//   });

//   it('renders a heading', () => {
//     render(<Home />)

//     const heading = screen.getByRole('heading', {
//       name: /welcome to next\.js!/i,
//     })

//     expect(heading).toBeInTheDocument()
//   })
// });
