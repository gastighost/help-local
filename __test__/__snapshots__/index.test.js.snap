// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`insert renders homepage unchanged 1`] = `
<div>
  <div
    class="container"
  >
    <main>
      <h2
        class="subtitle"
      >
        You are NOT connected to MongoDB. Check the 
        <code>
          README.md
        </code>
         
        for instructions.
      </h2>
      <a
        href="/jobs"
      >
        <div
          class="carded"
        >
          <h2>
            Jobs
          </h2>
        </div>
      </a>
      <a
        href="/education"
      >
        <div
          class="carded"
        >
          <h2>
            Education
          </h2>
        </div>
      </a>
      <a
        href="/humanitarian-aid"
      >
        <div
          class="carded"
        >
          <h2>
            Humanitarian Aid
          </h2>
        </div>
      </a>
      <a
        href="/social-media"
      >
        <div
          class="carded"
        >
          <h2>
            Social Media
          </h2>
        </div>
      </a>
    </main>
  </div>
</div>
`;
